test/nexus_test.go

```diff
@@ -302,7 +302,8 @@ func TestNexusSyncOperation(t *testing.T) {
 		_, err := nexus.ExecuteOperation(ctx, nc, syncOp, "already-started", nexus.ExecuteOperationOptions{})
 		var handlerErr *nexus.HandlerError
 		require.ErrorAs(t, err, &handlerErr)
-		require.Equal(t, nexus.HandlerErrorTypeBadRequest, handlerErr.Type)
```

Comment:
Don't you need to flag these if you are not using the prerelease test server?

Reply:
Is this for the cloud tests? I see that the integration tests are passing (apart from flakes).

Reply:
Saw the CI failures, added a flag to skip.